doc/source/changelog_generator.rst

```diff
@@ -0,0 +1,68 @@
+.. _changelog_generator:
+
+Generating Change Logs
+======================
+
+`Keg` comes with a separate tool that can be used to produce a change log
+for a generated image description from the git commit history of the used
+`keg recipes` tree(s). This obviously requires these `keg recipes` to be
+stored in a git repositories.
+
+For the change log generator to work, the image description for which the
+change log(s) need to have been generated with source info tracking, which
+can be enabled using the `-s` command line switch.
```

Comment:
This sentence sounds bumpy. I read it 3 times and still don't know 100% what it tries to tell me.

Reply:
I've reworded it. Let me know if it's still unclear.